guides/common/attributes-base.adoc

```diff
@@ -111,3 +111,5 @@
 :SmartProxyServerTitle: Smart{nbsp}Proxy{nbsp}Server
 :Team: Foreman developers
 :sectanchors:
+:pkg_arch: noarch
+:pkg_type: rpm
```

Comment:
Isn't this rather the package (file) extension?

Reply:
Renamed.